docs/l4-l7.rst

```diff
@@ -479,3 +515,38 @@ Dynamic Endpoint Discovery functionality is provided by F5ACIServiceCenter and h
 To view pool members after share node option is disabled for an existing partition user need to sync endpoint manually to BIG-IP by Application inventory → Partition → Sync EP icon
 
 ------
+
+**Q.Why older partitions that no longer exist on the BIG-IP are still retrieved by the AS3 endpoint?**
+
+The L4-L7 Application Inventory tab in FASC Application displays a list of all FAST and AS3 applications created on the BIG-IP device as well as on the FASC application. The Application Inventory tab on FASC always pulls the current AS3 configurations present on the BIG-IP device and it gets updated whenever the tab is clicked or reloaded. However in case of misconfigurations or configuration failures on the BIG-IP, users might see certain FAST/AS3 applications displaying on the Application Inventory tab which no longer exist on the BIG-IP.  If you run into this condition,  you can do a GET request through Postman on the following AS3 Endpoint API to verify if the concerned partition is still present on the BIG-IP in the API response.
+
+API URL: https://<BIGIP-IP:Port-Here>/mgmt/shared/appsvcs/declare?showHash=true
+Request type: GET
+Authentication type: Basic Auth (Enter username & password of the BIG-IP here)
+
+Hence, If the concerned partition is present in the API response, please report this issue/bug with the F5 BIG-IP AS3 team.  If the concerned partition is not present in the API response, please report this issue with F5 FASC team at https://github.com/F5Networks/f5-aci-servicecenter/issues
```

Comment:
Therefore, if the relevant partition is present in the API response, report this issue...
If the relevant partition is not present in the API response, report this issue....